.changeset/orange-coins-end.md

```diff
@@ -0,0 +1,18 @@
+---
+"@igloo-ui/tag": major
+"@igloo-ui/tag-picker": major
+---
+
+Renamed the appearance prop values for the tag component and removed the original default appearance. It now takes on the old "grey" appearance.
+
+## BREAKING CHANGE
+
+Anywhere Tag uses the appearance tag, it'll need to be renamed like the following:
```

Comment:
I feel like we should do something to ease the transition, instead of requiring every instance to be updated.
Like
v2.0 - We support new appearance values, old values are remapped to new values
v2.1 - We add a warning on old values when they are detected, with a timeline for deprecation
v2.2 - We deprecate old values, throw exception when they are detected
v2.3 - Remove detection of old values